common/lib/xmodule/xmodule/tests/test_graders.py

```diff
@@ -43,14 +43,14 @@ def test_weighted_grading(self):
         agg_fields['first_attempted'] = now
         scores.append(ProblemScore(weighted_earned=3, weighted_possible=5, graded=True, **prob_fields))
         all_total, graded_total = aggregate_scores(scores)
-        assert round(all_total - AggregatedScore(tw_earned=3, tw_possible=10, graded=False, **agg_fields), 7) >= 0
-        assert round(graded_total - AggregatedScore(tw_earned=3, tw_possible=5, graded=True, **agg_fields), 7) >= 0
+        assert all_total == AggregatedScore(tw_earned=3, tw_possible=10, graded=False, **agg_fields)
+        assert graded_total == AggregatedScore(tw_earned=3, tw_possible=5, graded=True, **agg_fields)
 
         # (0/5 non-graded) + (3/5 graded) + (2/5 graded) = 5/15 total, 5/10 graded
         scores.append(ProblemScore(weighted_earned=2, weighted_possible=5, graded=True, **prob_fields))
         all_total, graded_total = aggregate_scores(scores)
-        assert round(all_total - AggregatedScore(tw_earned=5, tw_possible=15, graded=False, **agg_fields), 7) >= 0
-        assert round(graded_total - AggregatedScore(tw_earned=5, tw_possible=10, graded=True, **agg_fields), 7) >= 0
+        assert all_total == AggregatedScore(tw_earned=5, tw_possible=15, graded=False, **agg_fields)
+        assert graded_total == AggregatedScore(tw_earned=5, tw_possible=10, graded=True, **agg_fields)
```

Comment:
not sure if doing `==` is the best thing to replace `unittest`'s  `assertAlmostEqual`, it should've been used if we had `assertEqual`

Reply:
Yeah, I am also not clear why it was being used. The checks above were using `assertEqual()`. Also, `AggregatedScore` does not [implement](https://github.com/edx/edx-platform/blob/9ab240f87e4a64f588f202bffcbf70c9b2ed6683/common/lib/xmodule/xmodule/graders.py#L89-L106) `__abs__()` so the only way `assertAlmostEqual()` would have worked is if the first `==` was true [here](https://github.com/python/cpython/blob/024325db5849c2cf1aa3b4c1013f73549712904e/Lib/unittest/case.py#L856-L862).